using-mobile-extensions/adobe-target/README.md

```diff
@@ -542,9 +542,21 @@ When merging, the new keys in the mbox parameters or the profile parameters are
 
 ## Target sessions
 
-The Target extension \(version 2.1.4 for iOS\) and \(version 1.1.3 for Android\) now supports persistent sessions. When a Target request is received, if a session ID does not exist, a new ID is generated and is sent in the request. This ID, with the Edge Host that is returned from Target, is kept in persistent storage for the configured `target.sessionTimeout` period. If the timeout value is not configured, the default value is 30 minutes.
```

Comment:
is the escape not needed?  it looks correct on live.

Reply:
Yeah it's not needed (as per Caleb) and I've been removing it from the parts of the doc that I update.

Reply:
Yeah, the escapes aren't necessary.

It's more, for me, that it looks ugly aesthetically on the markdown side, so it can be removed.

Sometimes escapes _may_ be necessary, but in this instance, we don't need them.